backend/src/v1/routes/wordRouter.ts

```diff
@@ -1,17 +1,17 @@
 import { Router } from "express";
 
-import { WordController } from "../controllers/wordController";
+import { ProvideWordController, WordController } from "../controllers/wordController";
 
 export const WordRouter = Router();
```

Comment:
try to stick to a naming convention. Usually objects are `camelCase` and classes/types are `PascalCase`. So this should be `wordRouter`.